### PR TITLE
Paginate the backups page

### PR DESCRIPTION
**Pull request by jvanbaarsen**
Some apps have a lot of backups, and since we do not paginate anything, the load
times are terrible. By adding pagination we'll fix this problem

### CODE REVIEW

**Comment by joshuajansen** [comment]
Nice 👌  How does the default html that kaminari generates look with Bulma?

**Comment by jvanbaarsen** [comment]
@joshuajansen Awesome:
image

**Comment by joshuajansen** [comment]
Ooh sweet! :shipit:

File: app/views/backups/index.html.erb [hunk 1]
```diff
@@ -20,6 +20,7 @@
         <%= render(@backups) || render("no_backups") %>
       </tbody>
     </table>
+    <%= paginate @backups %>
```
Comment (brambokdam) [inline app/views/backups/index.html.erb]: Do we need styling for this?

Comment (brambokdam) [inline app/views/backups/index.html.erb]: Never mind :)

Comment (jvanbaarsen) [inline app/views/backups/index.html.erb]: To late to the party, see discussion with @joshuajansen :trollface:

**Pull request merged**